### ISSUE TITLE
Add ability to add custom modules to the bar

### ISSUE THREAD

**Issue by RhapsodySL** [Mikeprod/XIV_Databar#99]
This doesn't need to be particularly complicated. Ideally, all that would be needed is the ability to make a button run a custom command, a few preset appearances for the button (maybe a resources folder to add your own icons?), and the ability to choose where specifically in the bar it goes (between existing icons?).

**Comment by AmenitradieDruidin** [comment]
Yes, I had already expressed this wish. That would be just great. Doesn't even have to be with an icon. 
Eg I'd like to show me the invasion of this addon. It works great with other data bars. But I don't want to use any other Databar. I love XIV_databar. That would be the Addon InvasionTimer: https://www.curseforge.com/wow/addons/invasiontimer
